core/domain/state_domain.py

```diff
@@ -797,7 +797,11 @@ class WrittenTranslations(python_utils.OBJECT):
     """
 
     def __init__(self, translations_mapping):
-        """Initializes a WrittenTranslations domain object."""
+        """Initializes a WrittenTranslations domain object.
+
+        Args:
+            translations_mapping: WrittenTranslations domain object.
```

Comment:
Are you sure about the type of `translations_mapping`? Also, follow the [style guide](https://github.com/oppia/oppia/wiki/Coding-style-guide#python) for writing docstring.

Reply:
I really do not know what type of translations_mapping it is. I reviewed the code but still couldn't decipher it.

Reply:
Maybe [this line](https://github.com/oppia/oppia/pull/7741/files#diff-fddf42bff012423be27edc64edd9e4e9L844) will be helpful?

Reply:
@DubeySandeep it actually was thanks.

Reply:
Done!!!!